packages/dashboard-frontend/stories/widgets/organic-sessions-widget/get-compare-data.js

```diff
@@ -0,0 +1,25 @@
+import { getRandom } from "./get-random";
+
+/**
+ * @typedef {"current"|"previous"} OrganicSessionsPeriod The organic sessions period.
+ */
+
+/**
+ * @typedef {Object<OrganicSessionsPeriod, Object<"sessions",number>>} RawOrganicSessionsCompareData The organic sessions compare data.
+ */
+
+/**
```

Comment:
It would be nice to comment here that this is a function for mocking data for the stories.